src/renderer/api/endpoints/metrics.api.ts

```diff
@@ -78,10 +78,24 @@ export function normalizeMetrics(metrics: IMetrics, frames = 60): IMetrics {
       result.forEach(res => {
         if (!res.values || !res.values.length) return;
 
+        let now = moment().startOf("minute").subtract(1, "minute").unix();
+        let timestamp = res.values[0][0];
+
+        while (timestamp <= now) {
+          timestamp = moment.unix(timestamp).add(1, "minute").unix();
+
+          if (!res.values.find((value) => value[0] === timestamp)) {
+            res.values.push([timestamp, "0"]);
+          }
+        }
```

Comment:
I still don't see how this fixes the issue. If `timestamp` is not `startOf("minute")` then line 87 will always lead to line 88.

Maybe I am not fulling understanding exactly what the goal is. But I definitely agree with @aleksfront that there should have been tests added with this change.